include/linker/linker-defs.h

```diff
@@ -249,6 +249,16 @@ extern char _vector_end[];
 /* end address of image, used by newlib for the heap */
 extern char _end[];
 
+/* Memory owned by the kernel. Start and end will be aligned for memory
```

Comment:
Is this really owned by the kernel? @SebastianBoe ?

Reply:
Assuming he's talking about kernelspace, no, it is not in kernelspace.

Reply:
So in your opinion, who is the owner?

Reply:
I don't understand the question. Can you define owner?

It might be that we would want to say "Memory managed by Zephyr". Or just omit the statement, as most of the memory is managed by zephyr.

If you could try to explain what you want to say, then maybe I could help with how to say it :)

Reply:
The memory is doesn't owned by application, but if say it is managed by Zephyr, it's hard to understand, I think.

How about:
Memory are layout by the linker based on the MPU's requests.

Reply:
That sounds a bit better ...

But maybe:

Memory in the 'ramfunc' sections are placed by the linker based on the MPU's requirements.

Reply:
Thanks, changed.